package.json

```diff
@@ -94,7 +95,8 @@
     "shared": "workspace:*",
     "slugify": "^1.6.5",
     "suspense": "^0.0.53",
-    "use-context-menu": "0.4.13"
+    "use-context-menu": "0.4.13",
+    "zod": "^3.23.6"
```

Comment:
Btw it looks like you added an unused dependency here (`zod`)

Reply:
That's something that dub requires. It might need to be a peer dep or something.